tests/Client/UpdaterTest.php

```diff
@@ -277,6 +277,20 @@ public function testVerifiedDelegatedDownload(): void
                 'level_3' => 1,
                 'level_3_below_terminated' => 1,
             ],
+            // Roles after terminating delegation where the target path does not
```

Comment:
"...where the target path does not the terminating role are..." seems like odd phrasing. Missing a word?

Reply:
Added more context. Take another look